app/controllers/secrets_controller.rb

```diff
@@ -1,5 +1,5 @@
 class SecretsController < ApplicationController
-  before_filter :require_user, :load_autofingers, except: [:new, :create]
+  before_filter :require_user, :load_autofingers
```

Comment:
I think the current code allows people to post secrets without being logged in (for anyone worried about being tracked). We'll probably want to keep that capability?